core/src/test/java/org/elasticsearch/explain/ExplainActionTests.java

```diff
@@ -138,7 +139,7 @@ public void testExplainWithFields() throws Exception {
         assertThat(response.getExplanation().getValue(), equalTo(1.0f));
         assertThat(response.getGetResult().isExists(), equalTo(true));
         assertThat(response.getGetResult().getId(), equalTo("1"));
-        assertThat(response.getGetResult().getFields().size(), equalTo(1));
+        assertThat(response.getGetResult().getFields().size(), greaterThanOrEqualTo(1));
```

Comment:
Why is this `greaterThanOrEqualTo()`? Do we not know how many metadata fields we should get returned here?